UnityProject/Assets/Scripts/Objects/ExecutionDevices/ExecutionDeviceController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Globalization;
+using AdminTools;
+using UnityEngine;
+
+namespace Objects.ExecutionDevices
+{
+	public class ExecutionDeviceController : MonoBehaviour, IRightClickable
+	{
+		private IExecutionDevice device;
+		public GameObject Victim { get; set; }
```

Comment:
clear this on destroy

Reply:
why?

Reply:
just in case it keeps a reference of it around, wwhen it's destroyed and not cleaned properly

Reply:
I've been unable to prove that this is an issue that can ever happen after testing and profiling a lot, but eh; sure.

Reply:
merge pls